Libraries/LibWeb/HTML/HTMLSelectElement.h

```diff
@@ -142,7 +144,7 @@ class WEB_API HTMLSelectElement final
     mutable Vector<GC::Ref<HTMLOptionElement>> m_cached_list_of_options;
     mutable size_t m_cached_number_of_selected_options { 0 };
 
-    GC::Ptr<HTMLOptionsCollection> m_options;
+    GC::Ptr<HTMLOptionsCollection> mutable m_options;
```

Comment:
We put `mutable` in front of the type, not after it. (See the lines above this one)